### ISSUE TITLE
hello i ummm cant get pass the loading screen

### ISSUE THREAD

**Issue by someaccidcabout** [juanmuscaria/a-mod-made-to-corrupt-a-well-known-block-game#4]
on your mod i cant get pass the loading screen. i just get stuck at some part of loading (it matters what device I'm on at time but it always something) and it just dont load at all. it is moving but that it. so plz fix this or help me fix this.

**Comment by yum13241** [comment]
Try a bunch of times and make sure that you aren't using the same seed over and over.

**Comment by someaccidcabout** [comment]
i dont think so but i dont think the seed bc even if i dont click the button that starts it it still does the same thing

**Comment by someaccidcabout** [comment]
ok now i trying it on my more powerful pc it just crashes at loading boot whatever resources

**Comment by someaccidcabout** [comment]
update 45: i got somewhat pass the loading boot whatever resources before the crash

**Comment by someaccidcabout** [comment]
ok so it freeze data now

**Comment by someaccidcabout** [comment]
i got pass it somehow????????

**Comment by someaccidcabout** [comment]
![how idk man](https://github.com/user-attachments/assets/2cf44e09-0229-411a-858a-b80fa815b478)
